### ISSUE TITLE
Non-option argument instead of --startup-url

### ISSUE THREAD

**Issue by tremby** [browsh-org/browsh#205]
Expected behaviour for me is to use the non-option arguments as URLs to load, but it seems I have to use the `--startup-url` option. I imagine I'm not the only one.

I'm a new user and not sure if you support tabs, but if you do, it'd make sense for multiple arguments to be URLs for multiple startup tabs.

**Comment by tobimensch** [comment]
I tend to agree.

**Comment by farhadmak** [comment]
I can work on this

**Comment by tobimensch** [comment]
@farhadmak 
Awesome. Thanks.

**Comment by tombh** [comment]
Sounds good to me too :)

**Comment by farhadmak** [comment]
Just want to confirm, we're wanting something like this?
`browsh www.google.com www.facebook.com`

**Comment by tobimensch** [comment]
@farhadmak 
That's exactly what we want.

**Comment by tremby** [comment]
Yeah, that looks to me like it'd open two tabs on startup, in that order. :)

**Comment by tobimensch** [comment]
@farhadmak 
I commented in your pull request #227 that I just merged, how this feature could be further refined. I'm hopeful to see more pull requests from you. Great work. I'm reopening this issue, because I consider this feature only partly implemented, as long as URLs such as lwn.net/path/to/article don't work.

**Comment by farhadmak** [comment]
Sounds good 👍
